components/clientComponents/globals/HydrationCheck.tsx

```diff
@@ -0,0 +1,12 @@
+"use client";
+import { useEffect, useState } from "react";
+
+export const HydrationCheck = () => {
+  const [hydrationComplete, setHydrationComplete] = useState(false);
```

Comment:
Wondering if we might want to rename this so it doesn't get mixed up with the template store hydration.

Reply:
Good idea.